### PR TITLE
[PHP 8.1] Calling a static element on a trait is deprecated

### PR DESCRIPTION
**Pull request by saundefined**
Deprecated Features [Changelog](https://www.php.net/manual/en/migration81.deprecated.php#migration81.deprecated.core.static-trait)

### CODE REVIEW

File: language/enumerations.xml [hunk 1]
```diff
@@ -486,6 +486,11 @@ enum Size
    They may only include methods and static methods.  A trait with properties will
    result in a fatal error.
   </para>
+  
+  <para>
+   As of PHP 8.1.0, calling static trait element directly is deprecated.
+   It should only be accessed on a class using the trait.
```
Comment (cmb69) [inline language/enumerations.xml:492]: This is about methods and properties, but you cannot *call* properties (you can *access* them). Unifying methods and properties as element is okay, but would need another verb (maybe "use"), but why not just duplicate the wording from the migration guide here?

Comment (saundefined) [reply language/enumerations.xml]: Thanks! Copied from the migration guide =)

**Review by cmb69 — APPROVED** [review]
Thank you!

**Comment by mumumu** [comment]
This deprecated feature is applied `only` to enumerations?
Should we add this to language/oop5/traits.xml ?

**Comment by saundefined** [comment]
@mumumu Oops, I guess I missed it, seems reasonable

**Comment by cmb69** [comment]
I think it should be *moved* to oop5/traits.xml.

**Pull request merged**